### PR TITLE
neomutt: Avoid creating empty files/directories

### PR DESCRIPTION
**Pull request by WhyNotHugo**
neomutt won't write to these locations. Processes it spawns might read
to some of them, but creating an empty file doesn't help. This just
pollutes user's $HOME with empty files and directories.

I've kept a few paths that MAY be written to by neomutt; it's not ideal,
but I want to minimise the risk of potential data loss, even if it is
corener cases.

See: https://github.com/netblue30/firejail/discussions/5276

### CODE REVIEW

**Review by kmk3 — APPROVED** [review]
LGTM.

Please do it for mutt.profile as well (either in this PR or in a new one).

**Comment by kmk3** [comment]
Related discussion from ~1.5 year ago about email client profiles polluting the
home directory and also about maintaining whitelists for the combinations of
text editors paths x email client profiles:

* <https://github.com/netblue30/firejail/pull/3849#issuecomment-752222345>

Misc: I think had made some changes similar to this PR locally, though I wasn't
sure about which paths to remove and the changes were done together with a big
refactoring (yak shaving, etc), which wasn't finished.

File: etc/profile-m-z/neomutt.profile [hunk 1]
```diff
@@ -50,31 +50,11 @@ include disable-programs.inc
...
 mkdir ${HOME}/Mail
 mkdir ${HOME}/mail
 mkdir ${HOME}/postponed
 mkdir ${HOME}/sent
-mkfile ${HOME}/.emacs
-mkfile ${HOME}/.mailcap
-mkfile ${HOME}/.msmtprc
-mkfile ${HOME}/.muttrc
-mkfile ${HOME}/.nanorc
-mkfile ${HOME}/.neomuttrc
```
Comment (rusty-snake) [inline etc/profile-m-z/neomutt.profile:74]: Who is `neomuttrc` supposed to be edited? Is there something like an "Open config file" options?

Comment (WhyNotHugo) [reply etc/profile-m-z/neomutt.profile:74]: > Who is neomuttrc supposed to be edited?

Usually it's created manually with a text editor.

> Is there something like an "Open config file" options?

Not that I know of. I think if you're very proficient you might figure out a way to possibly open neovim and then edit the config file manually, but that's _very_ unorthodox. I can't imagine someone doing this on a first run when no config file exists.

**Comment by WhyNotHugo** [comment]
Anything pending from me here?

**Comment by kmk3** [comment]
Merged, thanks!

**Pull request merged**